Three test failures on Python-3.5.0b4-amd64

Doesn't look serious:
```
--------------------------------------------------------------------
PIL SETUP SUMMARY
--------------------------------------------------------------------
version      Pillow 3.0.0.dev0
platform     win32 3.5.0b4 (v3.5.0b4:c0d641054635, Jul 26 2015, 07:11:12)
             [MSC v.1900 64 bit (AMD64)]
--------------------------------------------------------------------
--- TKINTER support available
--- JPEG support available
--- OPENJPEG (JPEG2000) support available (2.1)
--- ZLIB (PNG/ZIP) support available
--- LIBTIFF support available
--- FREETYPE2 support available
--- LITTLECMS2 support available
--- WEBP support available
--- WEBPMUX support available
--------------------------------------------------------------------
To check the build, run the selftest.py script.

--------------------------------------------------------------------
Pillow 3.0.0.dev0 TEST SUMMARY
--------------------------------------------------------------------
Python modules loaded from Pillow-git\PIL
Binary modules loaded from Pillow-git\PIL
--------------------------------------------------------------------
--- PIL CORE support ok
--- TKINTER support ok
--- FREETYPE2 support ok
--- LITTLECMS2 support ok
--- WEBP support ok
--- JPEG support ok
--- OPENJPEG (JPEG2000) support ok
--- ZLIB (PNG/ZIP) support ok
--- LIBTIFF support ok
--------------------------------------------------------------------
Running selftest:
--- 57 tests passed.

nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']
failed prepatch ... ok
succeeded prepatch ... ok
failed prepatch ... ok
succeeded prepatch ... ok
TestSanity.test_sanity ... ok
TestBinary.test_big_endian ... ok

<snip>

TestUtil.test_is_not_path ... ok
TestUtil.test_is_not_string_type ... ok
TestUtil.test_is_path ... ok
TestUtil.test_is_string_type ... ok

======================================================================
FAIL: TestFileMsp.test_open
----------------------------------------------------------------------
Traceback (most recent call last):
  File "Pillow-git\Tests\test_file_msp.py", line 34, in test_open
    self.assert_image_similar(im, hopper("1"), 4)
  File "Pillow-git\Tests\helper.py", line 105, in assert_image_similar
    ave_diff, epsilon))
AssertionError: 4.0 not greater than or equal to 6.12518310546875 :  average pixel value difference 6.1252 > epsilon 4.0000
-------------------- >> begin captured logging << --------------------
PIL.Image: DEBUG:
Traceback (most recent call last):
  File "Pillow-git\PIL\Image.py", line 2300, in open
    im = factory(fp, filename)
  File "Pillow-git\PIL\ImageFile.py", line 100, in __init__
    self._open()
  File "Pillow-git\PIL\ImImagePlugin.py", line 119, in _open
    raise SyntaxError("not an IM file")
  File "<string>", line None
SyntaxError: not an IM file
PIL.Image: DEBUG:
Traceback (most recent call last):
  File "Pillow-git\PIL\Image.py", line 2300, in open
    im = factory(fp, filename)
  File "Pillow-git\PIL\ImageFile.py", line 100, in __init__
    self._open()
  File "Pillow-git\PIL\ImtImagePlugin.py", line 44, in _open
    raise SyntaxError("not an IM file")
  File "<string>", line None
SyntaxError: not an IM file
PIL.Image: DEBUG:
Traceback (most recent call last):
  File "Pillow-git\PIL\Image.py", line 2300, in open
    im = factory(fp, filename)
  File "Pillow-git\PIL\ImageFile.py", line 100, in __init__
    self._open()
  File "Pillow-git\PIL\IptcImagePlugin.py", line 96, in _open
    tag, size = self.field()
  File "Pillow-git\PIL\IptcImagePlugin.py", line 76, in field
    raise SyntaxError("invalid IPTC/NAA file")
  File "<string>", line None
SyntaxError: invalid IPTC/NAA file
PIL.Image: DEBUG:
Traceback (most recent call last):
  File "Pillow-git\PIL\Image.py", line 2300, in open
    im = factory(fp, filename)
  File "Pillow-git\PIL\ImageFile.py", line 100, in __init__
    self._open()
  File "Pillow-git\PIL\MpegImagePlugin.py", line 71, in _open
    raise SyntaxError("not an MPEG file")
  File "<string>", line None
SyntaxError: not an MPEG file
PIL.PyAccess: DEBUG: New PyAccess: 1
PIL.PyAccess: DEBUG: {'image': 1021460806784, 'pixelsize': 1, 'mode': '1', 'image32': 0, 'block': 1021652173008, 'image8': 1021460806784, 'xsize': 128, 'linesize': 128, 'type': 0, 'palette': 0, 'depth': 0, 'destroy': 140733926045584, 'bands': 1, 'ysize': 128}
PIL.PyAccess: DEBUG: New PyAccess: 1
PIL.PyAccess: DEBUG: {'image': 1021460820304, 'pixelsize': 1, 'mode': '1', 'image32': 0, 'block': 1021652091008, 'image8': 1021460820304, 'xsize': 128, 'linesize': 128, 'type': 0, 'palette': 0, 'depth': 0, 'destroy': 140733926045584, 'bands': 1, 'ysize': 128}
--------------------- >> end captured logging << ---------------------

======================================================================
FAIL: TestImageGetColors.test_getcolors
----------------------------------------------------------------------
Traceback (most recent call last):
  File "Pillow-git\Tests\test_image_getcolors.py", line 22, in test_getcolors
    self.assertEqual(getcolors("P"), 90)  # fixed palette
AssertionError: 117 != 90
-------------------- >> begin captured logging << --------------------
PIL.PyAccess: DEBUG: New PyAccess: 1
PIL.PyAccess: DEBUG: {'image': 1021460823424, 'pixelsize': 1, 'mode': '1', 'image32': 0, 'block': 1021652091008, 'image8': 1021460823424, 'xsize': 128, 'linesize': 128, 'type': 0, 'palette': 0, 'depth': 0, 'destroy': 140733926045584, 'bands': 1, 'ysize': 128}
PIL.PyAccess: DEBUG: New PyAccess: L
PIL.PyAccess: DEBUG: {'image': 1021460832784, 'pixelsize': 1, 'mode': 'L', 'image32': 0, 'block': 1021652091008, 'image8': 1021460832784, 'xsize': 128, 'linesize': 128, 'type': 0, 'palette': 0, 'depth': 0, 'destroy': 140733926045584, 'bands': 1, 'ysize': 128}
PIL.PyAccess: DEBUG: New PyAccess: I
PIL.PyAccess: DEBUG: {'image': 1021460830704, 'pixelsize': 4, 'mode': 'I', 'image32': 1021460830704, 'block': 1021950615584, 'image8': 0, 'xsize': 128, 'linesize': 512, 'type': 1, 'palette': 0, 'depth': 0, 'destroy': 140733926045584, 'bands': 1, 'ysize': 128}
PIL.PyAccess: DEBUG: New PyAccess: F
PIL.PyAccess: DEBUG: {'image': 1021460833824, 'pixelsize': 4, 'mode': 'F', 'image32': 1021460833824, 'block': 1021950615584, 'image8': 0, 'xsize': 128, 'linesize': 512, 'type': 2, 'palette': 0, 'depth': 0, 'destroy': 140733926045584, 'bands': 1, 'ysize': 128}
PIL.PyAccess: DEBUG: New PyAccess: P
PIL.PyAccess: DEBUG: {'image': 1021460822384, 'pixelsize': 1, 'mode': 'P', 'image32': 0, 'block': 1021652091008, 'image8': 1021460822384, 'xsize': 128, 'linesize': 128, 'type': 0, 'palette': 102142223
9072, 'depth': 0, 'destroy': 140733926045584, 'bands': 1, 'ysize': 128}
--------------------- >> end captured logging << ---------------------

======================================================================
FAIL: TestImageHistogram.test_histogram
----------------------------------------------------------------------
Traceback (most recent call last):
  File "Pillow-git\Tests\test_image_histogram.py", line 12, in test_histogram
    self.assertEqual(histogram("1"), (256, 0, 10994))
AssertionError: Tuples differ: (256, 0, 10263) != (256, 0, 10994)

First differing element 2:
10263
10994

- (256, 0, 10263)
?            ^^^

+ (256, 0, 10994)
?            ^^^

-------------------- >> begin captured logging << --------------------
PIL.PyAccess: DEBUG: New PyAccess: 1
PIL.PyAccess: DEBUG: {'image': 1021460825504, 'pixelsize': 1, 'mode': '1', 'image32': 0, 'block': 1021652091008, 'image8': 1021460825504, 'xsize': 128, 'linesize': 128, 'type': 0, 'palette': 0, 'depth': 0, 'destroy': 140733926045584, 'bands': 1, 'ysize': 128}
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 661 tests in 88.860s

FAILED (SKIP=28, failures=3)

```

--- a/libImaging/Convert.c
+++ b/libImaging/Convert.c
@@ -998,7 +998,7 @@ frompalette(Imaging imOut, Imaging imIn, const char *mode)
     return imOut;
 }
 
-#if defined(_MSC_VER) && (_MSC_VER == 1600)
+#if defined(_MSC_VER)
 #pragma optimize("", off)
 #endif
 static Imaging
@@ -1233,7 +1233,7 @@ tobilevel(Imaging imOut, Imaging imIn, int dither)
 
     return imOut;
 }
-#if defined(_MSC_VER) && (_MSC_VER == 1600)
+#if defined(_MSC_VER)
 #pragma optimize("", on)
 #endif
 

